src/main/java/net/minecraftforge/server/ForgeTimeTracker.java

```diff
@@ -1,129 +0,0 @@
-/*
- * Minecraft Forge
- * Copyright (c) 2016.
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation version 2.1
- * of the License.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- */
-
-package net.minecraftforge.server;
-
-import java.lang.ref.WeakReference;
-import java.util.Arrays;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableMap.Builder;
-import com.google.common.collect.MapMaker;
-
-import net.minecraft.entity.Entity;
-import net.minecraft.tileentity.TileEntity;
-
-public class ForgeTimeTracker {
```

Comment:
It's not really safe to move this file. Mods can reference it and would break when this is removed. 
To keep compatibility, deprecate this class with a note to remove it in 1.13, and have its public methods forward to the new file.

Reply:
Do you want me to deprecate the public fields, such as `tileEntityTracking` and `tileEntityTrackingTime`?

Reply:
For good measure, sure.